wp-parsely.php

```diff
@@ -73,6 +73,7 @@
 require_once __DIR__ . '/src/UI/class-metadata-renderer.php';
 require_once __DIR__ . '/src/Endpoints/class-metadata-endpoint.php';
 require_once __DIR__ . '/src/Endpoints/class-graphql-metadata.php';
+require_once __DIR__ . '/src/Telemetry/telemetry-backend.php';
```

Comment:
Just asking, not proposing something specific. Do we think it is good to leave the name as-is?

Reply:
That's a pretty good point. I have considered here the word `backend` as the backend being used by the Telemetry module. But now that you mention it, it doesn't make much sense. I think I will rename it to `telemetry-init.php`